common/build.gradle

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2023 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+plugins {
+    id "java-library"
+    id "checkstyle"
+}
+
+dependencies {
```

Comment:
nit - i think you can remove the entire dependencies {} section if there's nothing there ?

Reply:
removed